webserver/restapi.py

```
@@ -1,27 +1,150 @@
     _handler_callback_post = callback
     print("POST Callback registered successfully for rest_blueprint!")
 
+
+@restapi_bp.route("/users", methods=["POST"])
+def create_user():
+    # check if there are any users in the database
+    users_exist = User.query.first() is not None
+
+    # if there are no users, we don't need to verify JWT
+    if users_exist:
+        if verify_jwt_in_request(optional=True) is None:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)